src/main/resources/chromedriver.properties

```diff
@@ -0,0 +1,4 @@
+# ----- ----- <chrome driver configuration> ----- -----
+chromedriver.path.unix=/src/main/java/driver/chromedriver/unix/chromedriver
```

Comment:
Maybe will be better move this files in another folder?

Reply:
Was fix chromedriver path in project structure: https://github.com/alexace013/my-web-site-automation/commit/a9b7659d700db9cc44bcef126cd1c969c5142490